code/datums/datacore.dm

```diff
@@ -72,25 +76,30 @@ GLOBAL_DATUM_INIT(datacore, /datum/datacore, new)
 		if(!department && !(name in heads) && (rank in GLOB.jobs_regular_all))
 			misc[name] = rank
 	if(length(heads) > 0)
-		dat += "<tr><th colspan=3>Command Staff</th></tr>"
+		dat += "<tr><th colspan=3>Command And Control Unit</th></tr>"
 		for(var/name in heads)
 			dat += "<tr[even ? " class='alt'" : ""]><td>[name]</td><td>[heads[name]]</td><td>[isactive[name]]</td></tr>"
 			even = !even
+	if(length(support) > 0)
+		dat += "<tr><th colspan=3>Auxiliary Support Staff</th></tr>"
+		for(var/name in support)
+			dat += "<tr[even ? " class='alt'" : ""]><td>[name]</td><td>[support[name]]</td><td>[isactive[name]]</td></tr>"
+			even = !even
 	if(length(mar) > 0)
-		dat += "<tr><th colspan=3>Marines</th></tr>"
+		dat += "<tr><th colspan=3>Marine Personnel</th></tr>"
 		for(var/j in list("Alpha","Bravo","Charlie", "Delta"))
 			dat += "<tr><th colspan=3>[j]</th></tr>"
```

Comment:
hide empty squads

Reply:
That... is something I cannot really do, sadly.